src/boot/stylesheets/main.scss

```diff
@@ -44,6 +44,18 @@ body {
     	color: $gray;
     }
 
+    button {
+		background-color: transparent;
```

Comment:
> I guess how do we feel about links that perform actions?

Is your definition of a "link" when the trigger is just text? I guess I've always thought that a button triggers an action (being just text or not), while an anchor/link sends us somewhere in the browser.

> While it's not ideal, it also feels strange that we're setting some base styling for buttons to look like links.

The unfortunate thing about the `<button>` element is that it's heavily styled by default in browsers, and seems like it always needs a certain amount of reset:

![screen shot 2017-06-06 at 3 13 46 pm](https://user-images.githubusercontent.com/349751/26854384-caf57f26-4aca-11e7-90a0-96460ce458fa.png)